contrib/seekable_format/zstdseek_decompress.c

```diff
@@ -433,6 +433,11 @@ size_t ZSTD_seekable_initAdvanced(ZSTD_seekable* zs, ZSTD_seekable_customFile sr
 
 size_t ZSTD_seekable_decompress(ZSTD_seekable* zs, void* dst, size_t len, unsigned long long offset)
 {
+    unsigned long long eos = zs->seekTable.entries[zs->seekTable.tableLen].dOffset;
```

Comment:
nit: this code base is very `const` friendly.
=> `unsigned long long const eos = ..`